app/cli/main.go

```diff
@@ -16,25 +16,82 @@
 package main
 
 import (
+	"crypto/sha256"
+	"fmt"
 	"os"
+	"strings"
 
 	"github.com/chainloop-dev/chainloop/app/cli/cmd"
+	"github.com/chainloop-dev/chainloop/app/cli/internal/telemetry"
+	"github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/posthog"
 	v1 "github.com/chainloop-dev/chainloop/app/controlplane/api/controlplane/v1"
 	"github.com/go-kratos/kratos/v2/errors"
 	jwtMiddleware "github.com/go-kratos/kratos/v2/middleware/auth/jwt"
 	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	"google.golang.org/grpc/status"
 )
 
+// Follow the convention stated on https://consoledonottrack.com/
+const doNotTrackEnv = "DO_NOT_TRACK"
+
+var (
+	// Posthog API key and endpoint to be injected in the build process
+	posthogAPIKey   = ""
+	posthogEndpoint = ""
+)
+
 func main() {
 	// Couldn't find an easier way to disable the timestamp
 	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr, FormatTimestamp: func(interface{}) string { return "" }})
 	rootCmd := cmd.NewRootCmd(logger)
 
-	if err := rootCmd.Execute(); err != nil {
-		msg, exitCode := errorInfo(err, logger)
+	// Set the default exit code to 0, in case of an error it will be updated
+	exitCode := 0
```

Comment:
nitpick. Leveraging default values would have helped in this case too. i.e `var exitCode int`

Reply:
Sure.